src/store/store.js

```diff
@@ -188,5 +84,58 @@ export default new Vuex.Store({
         router.push({ name: 'Login' });
       }
     },
+
+    async getColumns({ commit }) {
+      const response = await axios.get(`${process.env.VUE_APP_BASE_URL}/api/column`);
+      commit('setColumns', response.data);
+    },
+    async deleteColumn({ commit }, { id }) {
+      await axios.delete(`${process.env.VUE_APP_BASE_URL}/api/column`, {
+        id,
+      });
+      commit('deleteColumn', id);
+    },
+    async addColumn({ commit }, { name }) {
+      const response = await axios.post(`${process.env.VUE_APP_BASE_URL}/api/column`, {
+        name,
+      });
+      commit('addColumn', response.data);
+    },
+    async deleteTask({ commit }, { columnId, taskId }) {
+      await axios.delete(`${process.env.VUE_APP_BASE_URL}/api/column/task`, {
```

Comment:
to send body with request via axios, you should use axios.delete(URL, { data: body }), but not just axios.delete(URL, body)